agent/handlers/task_server_setup_test.go

```diff
@@ -1675,6 +1685,58 @@ func TestV4ContainerAssociation(t *testing.T) {
 	assert.Equal(t, expectedAssociationResponse, string(res))
 }
 
+func TestGQLMetadata(t *testing.T) {
+	testCases := []struct {
+		name                 string
+		gqlCreateSchemaError error
+		statusCode           int
+		expectedResponse     string
+	}{
+		{
+			"Create schema Error",
```

Comment:
nit. As we have 4 entries here, could we reformat it to 
```
{
    name:  "Create schema Error",
    gqlCreateSchemaError: errors.New("🤬"),
    statusCode: http.StatusInternalServerError,
    expectedResponse: expectedFailedGQLResponse
}
```
for better readability? Thanks!